docs/building-a-form/about-the-schema-and-uischema-objects.md

```diff
@@ -184,10 +184,14 @@ The us-forms-system code includes additional `uiSchema` functionality not found
       'value': <p>Some text</p>
     },
 
-    // A string of class names that are added to the widget for the current field.
+    // A list of class names that are added to the widget for the current field.
```

Comment:
I appreciate the docs update! Since the documentation on this option applies to several widgets I think they all should be changed for consistency. The other references are in [CurrencyWidget](https://github.com/usds/us-forms-system/blob/95c6518d5a0cc3a40f7376840760026b16007e13/src/js/widgets/CurrencyWidget.jsx#L50), [SelectWidget](https://github.com/usds/us-forms-system/blob/95c6518d5a0cc3a40f7376840760026b16007e13/src/js/widgets/SelectWidget.jsx#L37), and [ArrayCountWidget](https://github.com/usds/us-forms-system/blob/95c6518d5a0cc3a40f7376840760026b16007e13/src/js/widgets/ArrayCountWidget.jsx#L72). Can you update those as well?  I'm fine with just having the existing test on TestWidget for class names rather than duplicating it for the other widgets.